source/_components/elkm1.markdown

```diff
@@ -42,18 +42,20 @@ There is currently support for the following device types within Home Assistant:
 
 ## {% linkable_title Configuration %}
 
-To integrate Elk-M1 controller with Home Assistant, add the following
+To integrate one or more Elk-M1 controllers with Home Assistant, add the following
 section to your `configuration.yaml` file:
 
 ```yaml
 # Example configuration.yaml entry
 elkm1:
-  host: elk://IP_ADDRESS
+  - host: elk://IP_ADDRESS_1
+  ...
```

Comment:
Add a prefix for one of the controllers.

Reply:
I updated my patch-2 branch to address your feedback, @MartinHjelmare.  The configuration section isn't rendering correctly on github and I don't know why.

Reply:
This is due to some recent change and this branch is a bit behind. Nevertheless, tested it locally and works fine. So I'm making the call to merge it in.